ui/app/components/app/connected-sites-list/connected-sites-list.container.js

```diff
@@ -1,52 +1,16 @@
 import { connect } from 'react-redux'
-
 import ConnectedSitesList from './connected-sites-list.component'
-import {
-  showModal,
-  legacyExposeAccounts,
-  getOpenMetamaskTabsIds,
-} from '../../../store/actions'
-import {
-  getRenderablePermissionsDomains,
-  getPermissionsDomains,
-  getSelectedAddress,
-  getPermittedAccountsForCurrentTab,
-} from '../../../selectors/selectors'
-import { getOriginFromUrl } from '../../../helpers/utils/util'
+import { getOpenMetamaskTabsIds } from '../../../store/actions'
+import { getRenderablePermissionsDomains } from '../../../selectors/selectors'
 
 const mapStateToProps = (state) => {
-  const { openMetaMaskTabs } = state.appState
-  const { title, url, id } = state.activeTab
-  const permittedAccounts = getPermittedAccountsForCurrentTab(state)
-
-  let tabToConnect
-
-  if (url && permittedAccounts.length === 0 && !openMetaMaskTabs[id]) {
-    tabToConnect = {
-      title,
-      origin: getOriginFromUrl(url),
-    }
-  }
-
   return {
-    domains: getPermissionsDomains(state),
-    renderableDomains: getRenderablePermissionsDomains(state),
-    tabToConnect,
-    selectedAddress: getSelectedAddress(state),
+    connectedDomains: getRenderablePermissionsDomains(state),
   }
 }
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    showDisconnectAccountModal: (domainKey, domain) => {
-      dispatch(showModal({ name: 'DISCONNECT_ACCOUNT', domainKey, domain }))
-    },
-    showDisconnectAllModal: () => {
-      dispatch(showModal({ name: 'DISCONNECT_ALL' }))
-    },
-    legacyExposeAccounts: (origin, account) => {
-      dispatch(legacyExposeAccounts(origin, [account]))
-    },
     getOpenMetamaskTabsIds: () => dispatch(getOpenMetamaskTabsIds()),
```

Comment:
I'm not sure this belongs here anymore - it seems that the state this triggers a refresh of is now used exclusively in `ui/apps/pages/connected-sites/connected-sites.container.js`

Reply:
Moved around in 4991010